content/post/session4-startupdeepdive.md

```diff
@@ -0,0 +1,48 @@
+
+The 4th session of the Start-up Deep Dive series on Product Market Fit and GTM strategy was organized by IEDC MEC on 12 December at SDPK Hall, Model Engineering College. 
+
+
+**Mr. Thompson Tom** was in charge of the session. We could not have asked for a better speaker for this particular topic. Thompson Tom is the Founder and CEO of TutAR- Next Reality Classroom, which enables teachers to use digital teaching aids using augmented reality. It is the very first On-Screen AR app in the world. Thompson, a talented entrepreneur, got his first client (a school) to sign his product just 20 days after his MVP release. Today, TutAR is widespread in various parts of India and well-developed markets such as the US, Europe, and the Middle East.
+
+He started the session by introducing us to a few business terms and we learned about the importance of finding the right market for a product. Thompson emphasized the importance of building a powerful team who gives you honest reviews and creating a minimum viable product (MVP) to test the waters and gather valuable feedback from potential clients. 
+
+One key takeaway from the session was the importance of speaking to clients and pitching your product to the right target audience. By having open and honest communication with potential customers and getting your MVP into the hands of users, you can gather valuable insights into their needs and requirements, and understand what works and what doesn't. This information can be effectively utilized to fine-tune and iterate your product and ensure that it has got a strong market fit. 
+
+He also emphasized the importance of finding the right individuals who are passionate about your product and have the skills and experience necessary to help your company succeed. In his words, it's important not to get too caught up with planning. It is easy to get bogged down in discussions and debates about the perfect solutions, but the reality is that you won't know what works until you try it. And that's where the MVP comes in. It allows you to test your assumptions and gather real-world feedback that you can use to guide your product development. Thompson also reminded us that too often, teams can get stuck in the planning phase and spend too much time talking about their ideas rather than executing them. 
+
+Another important lesson from the session was the importance of understanding the different business models and how they can impact your product market fit. He discussed various business models like B2B, B2C, B2B2B, and B2B2C with suitable and familiar examples of successful companies that have used these models to explain each while keeping the session interactive. He also amplified the importance of choosing the right business model for your products while underscoring the fact that it’s okay to stick to more than one business model. By understanding the needs and preferences of your target market, you can determine which model is the best fit for your product.

```

Comment:
Can we actually move the second picture over here? 
Else it seems like a giant block of text stuffed b/w two images no?
The title of Gallery can be removed i think